docs/topics/cache.txt

```diff
@@ -548,6 +548,17 @@ flag on the connection's socket::
         }
     }
 
+``pymemcache`` by default sets the following options (you can override them on
+your :setting:`OPTIONS <CACHES-OPTIONS>`)::
+
+    'OPTIONS': {
+        'allow_unicode_keys': True,
+        # WARNING: changing default_noreply to True is not supported.
+        # See https://code.djangoproject.com/ticket/32749#comment:3
```

Comment:
Changing `default_noreply` breaks our test suite, but folks in general can do this. I would drop this comment.